website/docs/guides/whats-next.md

```diff
@@ -0,0 +1,21 @@
+# What's next?
+
+Congratulations! You have understood most core features of Docusaurus now. You have:
+
+- [Used the pages plugin](./creating-pages.md) to create a standalone React / Markdown page
+- [Used the docs plugin](./docs/docs-introduction.md) to create documentation pages. This includes [configuring the sidebar](./docs/sidebar/index.md), and even [versioning](./docs/versioning.md)
+- [Used the blog plugin](../blog.mdx) to create a fully featured blog
+- Tried your hands on [a range of Markdown features](./markdown-features/markdown-features-intro.mdx), which are useful for all content plugins
+- [Used stylesheets](../styling-layout.md) to customize your site's appearance
+- [Put images and other assets](../static-assets.md) in your pages
+- [Added search](../search.md) to your site
+- Understood how [browser support](../browser-support.md) and [SEO](../seo.md) are done through standard Docusaurus APIs
+- Learned about how [individual plugins](../using-plugins.md) are installed and configured
+- [Deployed](../deployment.mdx) your site to a content host
+- [Internationalized](../i18n/i18n-tutorial.md) your site to include multiple languages
+
+At this point, you probably have a big `docusaurus.config.js` already😄 However, you haven't written much code yet! Most of the features are implemented through calling encapsulated Docusaurus APIs. As you continue your journey, you can take three paths:
+
+- Learn more advanced Docusaurus concepts. This will help you gain a deeper understand of what the APIs do.
```

Comment:
how do we link to the advanced section? 🤪

Reply:
Mmm, I definitely planned to do that, probably forgot before the merge...